pkg/core/resources/apis/mesh/external_service_helpers_test.go

```diff
@@ -0,0 +1,32 @@
+package mesh
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+)
+
+var _ = Describe("ExternalServiceResource", func() {
+	Describe("MarshalLog", func() {
+		It("should marshal log without panicking", func() {
```

Comment:
I'd not call it like this, ideally `It("...")` should provide some value. Can we check the output of `MarshalLog` and check that it's indeed masked?

Reply:
changed